### PR TITLE
updated provider/temporal module, added ca cert var

### PR DESCRIPTION
**Pull request by michaelcramer**
updated aws provider to ~> 5, which also required an update of the temporal module.  confirmed deployment works correctly.
added a variable to allow control of which cert authority is used for rds.
removed provider block from main, per best practices.

### CODE REVIEW

**Review by roberto-retool — APPROVED** [review]

File: modules/aws_ecs/main.tf [hunk 1]
```diff
@@ -2,15 +2,11 @@ terraform {
...
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 5.0"
     }
   }
 }
 
-provider "aws" {
-  region = var.aws_region
```
Comment (anna-yn) [inline modules/aws_ecs/main.tf:11]: is this not needed anymore?

Comment (michaelcramer) [reply modules/aws_ecs/main.tf:11]: It's typically best practice to not have the provider block defined in a module.  Defining the block in the module can cause issues when trying to delete or comment out that module.

Comment (anna-yn) [reply modules/aws_ecs/main.tf:11]: i see! would changing this break existing usage?

**Comment by anna-yn** [comment]
oh i think you want to update the readme too https://github.com/tryretool/terraform-retool-modules/blob/main/README.md?plain=1#L8

File: modules/aws_ecs/main.tf (hunk 1 above)
Comment (michaelcramer) [reply modules/aws_ecs/main.tf:11]: it shouldn't, but i can test to be sure

Comment (michaelcramer) [reply modules/aws_ecs/main.tf:11]: tested various scenarios, did not break existing usage

**Review by anna-yn — APPROVED** [review]

**Pull request merged**